testing/src/test/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.android.testing">
   <uses-sdk android:minSdkVersion="19"
-    android:maxSdkVersion="30" />
+    android:targetSdkVersion="30" />
```

Comment:
Does this still require an android:maxSdkVersion to be specified? Ditto for data/src/test/AndroidManifest.xml.

Reply:
Not really. maxSdkVersion _can_ be useful to restrict libraries to certain versions, but it doesn't actually make sense in the sense of a sub-module of an app that's being actively developed (in fact, broadly a maximum SDK is really only useful when coupled with something meant for newer versions, e.g. to ensure people don't accidentally depend on the old version). I think this was just set up incorrectly originally and we missed it (since all other test manifests besides these two had it correctly configured before).